tests/album-delete.test.js

```diff
@@ -0,0 +1,74 @@
+const { expect } = require('chai');
+const request = require('supertest');
+const getDb = require('../src/services/db');
+const app = require('../src/app');
+
+describe('update Album', () => {
+  let db;
+  let artists;
+  let db2
+  let albums
```

Comment:
Why no semi colons at the end of each line here? As with indentation, coding standards for things like this should be consistent throughout the project